Multiple site plugin working

Boot-Error opened the pull request:
Modified `app.py`, multiple plugins can be called, fixing #13

Boot-Error committed:
Multiple site plugin working

amolkahat commented on requirement.txt:
```diff
@@ -1,2 +1,3 @@
 click>=5.1
 libpagure
+PyYaml
```
@Boot-Error Just remove the `PyYAML` from the file.

amolkahat commented on app.py:
```diff
@@ -1,8 +1,17 @@
-from  plugins.pagure import  get_pull_requests
+from pprint import pprint
 
+from plugins import github
```
@Boot-Error you can specify multiple packages in a single import.

Boot-Error committed:
Added decorator for plugins to safely show errors
Added FetchError to be used by plugins to raise Custom Fetch Errors
Added sys.exit for proper exit status!

Boot-Error committed:
Decorator for execute_board()

Boot-Error committed:
Merge branch 'master' into master

Boot-Error committed:
Merge branch 'master' into master

bhavin192 commented:
Please don't add other commits to the same PR, create one PR for one change.
